frontend/components/propositions/OccurrenceDetail.tsx

```diff
@@ -39,8 +40,13 @@ const OccurrenceDetail: FC<OccurrenceDetailProps> = ({ occurrence }: OccurrenceD
       {occurrence.summary != occurrence.title && <p className="lead">{occurrence.summary}</p>}
       <div dangerouslySetInnerHTML={{ __html: occurrence.elaboration }} />
       {occurrence.postscript && <p dangerouslySetInnerHTML={{ __html: occurrence.postscript }} />}
-      {occurrence.tagsHtml && (
-        <ul className="tags" dangerouslySetInnerHTML={{ __html: occurrence.tagsHtml }} />
+
+      {!!occurrence.cachedTags?.length && (
+        <ul className="tags">
+          {occurrence.cachedTags.map((topic) => (
+            <Tag key={topic.pk} topic={topic} />
+          ))}
+        </ul>
```

Comment:
As you mentioned, we should probably pull the move the [`li` in `Tag`](https://github.com/ModularHistory/modularhistory/pull/577/files#diff-37d2c9b2992d497dcd7baef41fb6d104295f3bd41075d8f745037bc7d8e045adR21) out here so the usage of `li` is coupled with `ul`.